openmc/deplete/msr.py

```diff
@@ -11,26 +11,28 @@
 from openmc.data import ELEMENT_SYMBOL
 
 class MsrContinuous:
-    """Class defining removal rates for continuous nuclides removal or feed.
+    """Class for defining continuous removals and feeds.
 
-    Molten Salt Reactor (MSR) benefits from continuously reprocessing the fuel
-    salt to remove fission products or to feed fresh fuel into the system. This
-    reactor category inspired the development of this class.
-    An instance of this class can be passed directly to an instance of one of the
-    integrator classes, such as :class:`openmc.deplete.CECMIntegrator`.
+    Molten Salt Reactors (MSRs) benefit from continuous reprocessing,
+    which removes fission products and feeds fresh fuel into the system. MSRs
+    inspired the development of this class.
 
-    .. versionadded:: 0.13.3
+    An instance of this class can be passed directly to an instance of one of
+    the :class:`openmc.deplete.Integrator` classes.
 
+    .. versionadded:: 0.13.3
     Parameters
     ----------
-    operator : openmc.Operator
-        Operator to perform transport simulations
+    operator : openmc.TransportOperator
+        Depletion operator
     model : openmc.Model
-        OpenMC Model object
+        OpenMC model containing materials and geometry. If using
+        :class:`openmc.deplete.CoupledOperator`, the model must also contain
+        a :class:`opnemc.Settings` object.
 
     Attributes
     ----------
-    burn_mats : list of str
+    burnable_mats : list of str
```

Comment:
`burnable_mats` must also be updated in `pool.py`.

Reply:
done